src/librustdoc/html/static/settings.js

```diff
@@ -32,6 +46,8 @@
                 toggle.onchange = function() {
                     changeSetting(this.id, this.checked);
                 };
+                toggle.onkeyup = handleKey;
+                toggle.onkeyrelease = handleKey;
```

Comment:
Doesn't this mean the handler will be called multiple times?

Reply:
The event callback is set to a given element, so only once per element. If you use the key but don't have focus on it, nothing will happen.